ee/clickhouse/queries/cohort_query.py

```diff
@@ -140,9 +139,12 @@ def __init__(
         self._earliest_time_for_event_query = None
         self._restrict_event_query_by_time = True
         self._cohort_pk = cohort_pk
-        self._cohorts_seen = cohorts_seen
+
+        # Simplify and make sure cohort properties are surfaced
```

Comment:
🤔 tried to do this but can't put the data through simplify multiple times otherwise the nesting will cause unintentional errors since we wrap props into their own groups during the first pass

Reply:
Feel free to remove these commits if you want to just merge the current changes

Reply:
Will do that, and merge this :)

Was the intent here to somehow simplify cohorts-within-cohorts to just cohorts, when possible? (and several edge cases along the line to make sure query building doesn't have additional joins and stuff?)